app/src/main/res/values/styles.xml

```diff
@@ -538,4 +538,26 @@
     <item name="android:clipToPadding">false</item>
   </style>
 
+  <style name="ChangeEmailPassTextInputEditText" parent="CalloutPrimary">
+    <item name="android:layout_width">match_parent</item>
+    <item name="android:layout_height">wrap_content</item>
+    <item name="android:layout_marginStart">@dimen/activity_horizontal_margin</item>
+    <item name="android:layout_marginEnd">@dimen/activity_horizontal_margin</item>
+  </style>
+
+  <style name="ChangeEmailPassTextInputEditTextMarginBottom" parent="CalloutPrimary">
```

Comment:
This can inherit from `ChangeEmailPassTextInputEditText` and then all you need is `<item name="android:layout_marginTop">@dimen/grid_1</item>`.

Also should you rename this to `ChangeEmailPassTextInputEditTextMarginTop`